target/linux/ipq806x/base-files/etc/init.d/bootcount

```diff
@@ -2,9 +2,9 @@
 
 START=99
 
-. "$IPKG_INSTROOT/lib/upgrade/asrock.sh"
-
 boot() {
+	. /lib/upgrade/asrock.sh
```

Comment:
This is probably just a workaround for some issue hidden in image generation code path.

Reply:
Maybe 1fdf6b745cc3d85be3743837817a360121554134 ?